src/main/archetype/pom.xml

```diff
@@ -303,7 +304,7 @@ Bundle-DocURL:
                 <plugin>
                     <groupId>org.apache.jackrabbit</groupId>
                     <artifactId>filevault-package-maven-plugin</artifactId>
-                    <version>1.0.3</version>
+                    <version>1.1.4</version>
                     <configuration>
                         <filterSource>src/main/content/META-INF/vault/filter.xml</filterSource>
```

Comment:
Please add the property to by default prevent deployment with cloud manager:

```
<properties>
          <cloudManagerTarget>none</cloudManagerTarget>
</properties>
```

And only enable deployment for the `all` container.